cypress/e2e/ui/Overview/reports.cy.js

```diff
@@ -0,0 +1,432 @@
+      expect(tableHeaders[2]).to.eq('E-Mail after Running');
+      expect(tableHeaders[3]).to.eq('To E-mail');
+      expect(tableHeaders[4]).to.eq('Report Filter');
+      expect(tableHeaders[5]).to.eq('Run At');
+      expect(tableHeaders[6]).to.eq('Last Run Time');
+      expect(tableHeaders[7]).to.eq('Next Run Time');
+      expect(tableHeaders[8]).to.eq('Zone');
+    });
+    cy.get('.content_value').then((values) => {
+      const nums = [...Array(values.length).keys()];
+      nums.forEach((index) => {
+        tableValues.push(values[index].innerText);
+      });
+    }).then(() => {
+      expect(tableValues[0]).to.eq('Cypress test schedule description');
+      expect(tableValues[1]).to.eq('True');
+      expect(tableValues[2]).to.eq('False');
+      expect(tableValues[3]).to.eq('');
+      expect(tableValues[4]).to.eq(reportFilter);
+      expect(tableValues[5]).to.contain(`${runHour}:${runMinute}:00`);
+      expect(tableValues[7]).to.contain(`${runHour}:${runMinute}:00`);
+      expect(tableValues[8]).to.eq('default');
+    });
+
+    // Click edit schedule
+    cy.get('#miq_schedule_vmdb_choice').click().then(() => {
+      cy.get('.bx--overflow-menu-options__btn').then((list) => {
+        cy.get(list.children()[0]).click();
+      });
+    });
+
+    // Edit the schedule information
+    reportFilter = '';
+    cy.get('#name').clear({ force: true }).type('Cypress Test Schedule Edit', { force: true });
+    cy.get('#description').clear({ force: true }).type('Cypress test schedule description edit', { force: true });
+
+    cy.get('#form_filter_div > .form-horizontal > :nth-child(1) > .col-md-8 > .btn-group > .btn').click({ force: true });
+    cy.get('#form_filter_div > .form-horizontal > :nth-child(1) > .col-md-8 > .btn-group > .open > .dropdown-menu > [data-original-index="0"] > a').click({ force: true });
+
+    cy.get('#form_filter_div > .form-horizontal > :nth-child(1) > .col-md-8 > .btn-group > .btn').click({ force: true });
+    cy.get('#form_filter_div > .form-horizontal > :nth-child(1) > .col-md-8 > .btn-group > .open > .dropdown-menu > [data-original-index="1"] > a').then((option) => {
+      cy.get(option).click({ force: true }).then(() => {
+        cy.wait(5000);
+        cy.get('#form_filter_div > .form-horizontal > :nth-child(2) > .col-md-8 > .btn-group > .btn').click({ force: true });
+        cy.get('#form_filter_div > .form-horizontal > :nth-child(2) > .col-md-8 > .btn-group > .open > .dropdown-menu > [data-original-index="1"] > a').then((option) => {
+          cy.get(option).click({ force: true }).then(() => {
+            cy.wait(5000);
+            cy.get(':nth-child(3) > .col-md-8 > .btn-group > .btn').click({ force: true });
+            cy.get(':nth-child(3) > .col-md-8 > .btn-group > .open > .dropdown-menu > [data-original-index="1"] > a').then((option) => {
+              cy.get(option).click({ force: true });
+              reportFilter = option[0].innerText;
```

Comment:
@Fryguy I know we don't want to use time based waits but I think these are necessary at least until this page gets converted to react. Currently there is no way to get the id for this schedule (no id in url or summary page) so there is no way to intercept the apis since they contain the schedule id. Once this page is converted to React the id would be contained in the url and these waits would no longer be needed but for now I think we should use them to improve test reliability. Without the waits the tests fail fairly inconsistently at this step due to timing issues in the form. With these wait statements it greatly improves reliability where the tests rarely fail here anymore.